src/java.net.http/share/classes/jdk/internal/net/http/HttpResponseImpl.java

```diff
@@ -44,6 +44,7 @@
 class HttpResponseImpl<T> implements HttpResponse<T>, RawChannel.Provider {
```

Comment:
The copyright year on this file will need an update.

Reply:
Fixed in 883135d8edc2f2a892152e42eda924d65686bc7a.